2.13/tests/test/test.js

```diff
@@ -465,23 +469,26 @@ function pollInstanceLastOperationStatus(instanceId, done) {
     var lastOperationState = 'in progress';
     async.whilst(
         function() {
-            return lastOperationState == "in progress" && count < config.maxPollingNum;
+            return lastOperationState == "in progress" && count <= config.maxPollingNum;
         },
         function(callback) {
             count++;
+            console.log(count.toString() + 'th polling last operation...');
             preparedRequest()
                 .get('/v2/service_instances/' + instanceId + '/last_operation')
                 .set('X-Broker-API-Version', apiVersion)
                 .auth(config.user, config.password)
                 .expect(200)
                 .expect('Content-Type', /json/)
                 .end(function(err, res){
-                    if (err) return done(err);
+                    if (err) callback(err, count);
                     var message = validateJsonSchema(res.body, lastOperationSchema);
-                    if (message != "") return done(new Error(message));
+                    if (message != "") return callback(new Error(message), count);

```

Comment:
When calling an error back, you can only call the error back as suffix function variables can be absent in Nodejs. And it is a habit in Nodejs, always putting the error in the first place and always checking if the error exists first in a callback.